rustls/src/webpki/verify.rs

```diff
@@ -278,13 +279,16 @@ impl WebPkiClientVerifier {
     /// client certificate validation.
     /// `revocation_check_depth` controls which certificates have their revocation status checked
     /// when `crls` are provided.
+    /// `unknown_revocation_policy` controls how certificates with an unknown revocation status
```

Comment:
How does this look in rustdoc? Should we use a bullet list here?

Reply:
Switched this and the server equivalent to use bulleted lists. They're both `pub(crate)` but it doesn't hurt to make the Rustdoc more readable for `--document-private-items`.